provider/internal/connector/resource_test.go

```diff
@@ -21,10 +21,10 @@ var testSdcResource = testSdcResourceType{
 }
 var testSdcResourceConfig = acctest.MustParseTemplate(testResourceTemplate, testSdcResource)
 
-var testResource_NewName = acctest.MustOverrideFields(testSdcResource, map[string]any{
-	"Name": "test-sdc-2",
-})
-var testResourceConfig_NewName = acctest.MustParseTemplate(testResourceTemplate, testResource_NewName)
+// var testResource_NewName = acctest.MustOverrideFields(testSdcResource, map[string]any{
```

Comment:
I guess it shouldn't be muted?

Reply:
I left it commented because there's a flake in this test and we need to investigate it further. https://jira-eng-rtp3.cisco.com/jira/browse/LH-70119

Reply:
ah kk